modules/timepoint_list/README.md

```diff
@@ -0,0 +1,48 @@
+# Timepoint List
+
+## Purpose
+
+The `timepoint_list` module serves as an entry-point for data entry
+about a candidate. It serves as a dashboard for a candidate linking
+to other modules that provide specific functionality about a candidate
```

Comment:
@driusan Missing period at end of sentence.

Reply:
thanks, fixed